src/DynamoManipulation/NodeManipulator.cs

```diff
@@ -88,6 +88,28 @@ internal IRenderPackageFactory RenderPackageFactory
         }
 
         internal Point3D? CameraPosition { get; private set; }
+        
+        // This is null for each new manipulator that is initialized 
+        private bool? enabled;
```

Comment:
nullable bool - scary!

Reply:
Having multiple boolean in this class getting enable state is confusing.. Would you consider getting enum state instead?

Reply:
Do you mean having an `Enabled` bool is confusing with an `IsEnabled()` method?

Reply:
Yes especially they mean different thing within the class, I do understand we can't touch the signature of `IsEnabled()` to simplify this. So I hope if we add a new property, we can return multiway states enums instead of  true/false/null. My two cents